website/tfe.erb

```diff
@@ -13,6 +13,9 @@
                 <li<%= sidebar_current("docs-tfe-datasource") %>>
                     <a href="#">Data Sources</a>
                     <ul class="nav nav-visible">
+                        <li<%= sidebar_current("docs-datasource-tfe-oauth-client") %>>
```

Comment:
i think this might need the -x at the end to match the sidebar_current value in the website/docs/d/oauth_client.html.markdown file. i think this not work as expected otherwise. 
you could also just remove the -x from the sidebar_current value in the website/docs/d/oauth_client.html.markdown file i think.